pkg/api/api.go

```diff
@@ -1,5 +1,12 @@
 package api
 
+const (
```

Comment:
I'm inclined to move these into the `configmap_builder.go` file, since they are only used in one place.